management/src/main/scala/akka/management/AkkaManagementSettings.scala

```diff
@@ -25,10 +25,23 @@ final class AkkaManagementSettings(val config: Config) {
 
     val Port: Int = {
       val p = cc.getInt("port")
-      require(p >= 0, s"akka.management.http.port MUST be > 0 (was ${p})")
+      require(0 to 65535 contains p, s"akka.management.http.port must be 0 through 65535 (was ${p})")
       p
     }
 
+    val BindHostname: String = cc.getString("bind-hostname") match {
+      case "" ⇒ Hostname
+      case value ⇒ value
+    }
```

Comment:
Hmm... This may be confusing, rather this should be EffectiveBindHostname where we have the logic perhaps; and the BindHostname should just be the "raw" value?

Reply:
Fixed